libmei/tools/parseschema2.py

```diff
@@ -56,7 +56,7 @@ def main(configure: dict) -> bool:
     p = ArgumentParser(usage='%(prog)s [-c config path] [compiled odd path]')
 
     p.add_argument("compiled", help="A compiled ODD file", type=Path)
-    p.add_argument("-c", "--config", help="Path to a config file", type=Path)
+    p.add_argument("-c", "--config", default="./config.yml", help="Path to a config file", type=Path)
```

Comment:
Now the default config is not working when running the script from the tools directory:
<img width="715" alt="FromTools" src="https://user-images.githubusercontent.com/63608463/219027047-28c3ed31-ba3d-4a53-ac01-2b23520d1b04.png">
When running it from `libmei` with default config, it works. I would still prefer to have more flexibility here.

Apart from this, it looks fine to me.

Reply:
So you would like the paths to resolve based on the parent directory of the `config.yml` file, rather than the `cwd` from where you are running the script?

Reply:
I think it would be nice to resolve the relative paths into reasonable absolute paths, i.e. the config default path should point to the location where the config file actually sits, independent of where the script is called from. Also relative paths in the config file should be considered relative to the config file location. I think using the `cwd` to resolve relative paths is not a good idea, in principle the `cwd` could be anything.

It's a nice-to-have, feel free to merge this PR without this.